apps/frontend/src/libs/components/menu/menu.tsx

```diff
@@ -3,13 +3,15 @@ import { IconButton, Popover } from "~/libs/components/components.js";
 import styles from "./styles.module.css";
 
 type Properties = {
+	buttonVariant?: "outlined" | "primary";
```

Comment:
Maybe it would be right to move this type to the other file, as we're using it in two places now, but hope that it's not a big deal, as we have only two variants.

Reply:
let's use `ComponentProps<typeof IconButton>` instead